src/Common/src/Interop/Windows/sspicli/NegotiationInfoClass.cs

```diff
@@ -17,14 +17,21 @@ internal class NegotiationInfoClass
 
         internal NegotiationInfoClass(SafeHandle safeHandle, int negotiationState)
         {
+            bool globalLogEnabled = GlobalLog.IsEnabled;
```

Comment:
We don't have to do it if you don't want to.  It's a minor optimization.  I don't think there's anything that can be done to simplify or improve IsEnabled though... it just does EventSource.IsEnabled().